manifests/autotune/layers/container-config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "apiVersion": "recommender.com/v1",
+  "kind": "KruizeLayer",
+  "metadata": {
+    "name": "container"
+  },
+  "layer_name": "container",
```

Comment:
Thanks for the PR @kusumachalasani 

I see the `name` in `metadata` section, should we be maintaining again as `layer_name` or they are different entities and it's a coincidence for both to be same in this scenario?